utilities/operator.py

```diff
@@ -218,7 +218,7 @@ def wait_for_mcp_updated_condition_true(machine_config_pools_list, timeout=TIMEO
 def wait_for_mcp_ready_machine_count(machine_config_pools_list):
     LOGGER.info("Waiting for MCPs to have all machines ready.")
     sampler = TimeoutSampler(
-        wait_timeout=TIMEOUT_10MIN,
+        wait_timeout=TIMEOUT_20MIN,
```

Comment:
Same comment here. If you need to increase value, please add an argument and use the original value as default.